core/tx_multicurrency_priceheap.go

```diff
@@ -10,9 +10,40 @@ import (
 
 type CurrencyCmpFn func(*big.Int, *common.Address, *big.Int, *common.Address) int
 
-// IsCheaper returns true if tx1 is cheaper than tx2 (GasPrice with currency comparison)
-func (cc CurrencyCmpFn) IsCheaper(tx1, tx2 *types.Transaction) bool {
-	return cc(tx1.GasPrice(), tx1.FeeCurrency(), tx2.GasPrice(), tx2.FeeCurrency()) < 0
+func (cc CurrencyCmpFn) GasTipCapCmp(tx, other *types.Transaction) int {
```

Comment:
Any particular reason for making those functions methods on `CurrencyCmpFn`? In my opinion it would be clearer to make those free functions, but that's just a nitpick.

Reply:
No reason, at the moment of writing it felt like it made sense. But I have no strong preference over it.